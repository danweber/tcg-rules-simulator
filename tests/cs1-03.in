VERSION:2
# "[Your Turn] When this Monster attacks, one of your opponent's Monsters get -2000 DP for the turn."
FIELD1:Attacker,DW1-Vanilla,CS1-03 Green,DW1-Birder
FIELD2:Vic0,CS1-02 Vic1,CS1-02,REST Vic2,DW1-Vanilla,REST
SECURITY2:CS1-02
HAND1:DW1-Standup
GAME:P1 T2 MAIN 5
DUMP:DP
ECHO: -- Attack once with Attacker, kill Vic0 with effect and Vic1 with attack
CMD:ATTACK Attacker Vic1
STEP:AUTOANSWER:CAPTURE
DUMP:DP
ECHO: -- This isn't once per turn! Restand and do it again. I only survive into Vic2 because it gets its DP reduced.
CMD:PLAY DW1-Standup
STEP:AUTOANSWER
CMD:ATTACK Attacker Vic2
DUMP:DP
