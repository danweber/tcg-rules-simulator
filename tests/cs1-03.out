FIELD1:Attacker,DW1-Vanillamon,CS1-03,5000DP Green,DW1-Birder,3000DP
FIELD2:Vic0,CS1-02,500DP Vic1,CS1-02,REST,500DP Vic2,DW1-Vanillamon,REST,5000DP
GAME:P1 T2 MAIN 5
ECHO: -- Attack once with Attacker, kill Vic0 with effect and Vic1 with attack
CAPTURE:Target Paper Shield 3
CAPTURE:Target Paper Shield 4
CAPTURE:Target VanillaMon 5
FIELD1:Attacker,DW1-Vanillamon,CS1-03,REST,5000DP Green,DW1-Birder,3000DP
FIELD2:Vic2,DW1-Vanillamon,REST,5000DP
GAME:P1 T2 MAIN 5
ECHO: -- This isn't once per turn! Restand and do it again. I only survive into Vic2 because it gets its DP reduced.
FIELD1:Attacker,DW1-Vanillamon,CS1-03,REST,5000DP Green,DW1-Birder,3000DP
FIELD2:
GAME:P1 T2 MAIN 1
